### PR TITLE
Fix Svace issues in webclient/webserver/websocket examples.

### PR DESCRIPTION
**Pull request by davidfather**
1. Use strncmp instead of strcmp
2. Delete duplicated usage print webclient example
3. Free malloced variables before a function returns

### CODE REVIEW

File: apps/examples/webclient/webclient_main.c [hunk 1]
```diff
@@ -253,16 +253,15 @@ int webclient_init_request(void *arg, struct http_client_request_t *request)
 		goto exit;
 	}
 
-	if (!strcmp(argv[1], "GET")) {
+	if (!strncmp(argv[1], "GET", 3)) {
```
Comment (junmin-kim) [inline apps/examples/webclient/webclient_main.c]: It seems strncmp("GETS","GET", 3) returns 0.
Is this intended?

Comment (davidfather) [reply apps/examples/webclient/webclient_main.c]: http(s) has four methods: GET, PUT, POST, DELETE, but GETS.

Comment (junmin-kim) [reply apps/examples/webclient/webclient_main.c]: Then, It should be if (!strncmp(argv[1], "GET", 4)), so only "GET" can enter the if statement

Comment (davidfather) [reply apps/examples/webclient/webclient_main.c]: I modify this as you mentioned.

**Pull request merged**